index.html

```diff
@@ -0,0 +1,686 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>The Next Web Clone</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/7290f27402.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header>
+        
+        <div class="navigation-first">
+            <div class="navigation-first-left">
+                <a href="#">News</a>
+                <a href="#">TNW2020</a>
+                <a href="#">Business</a>
+                <a href="#">AMAs</a>
+                <a href="#">Spaces</a>
+                <a href="#">Terms & Conditions</a>
+            </div>
+            <div class="navigation-first-right">
+                <a href="#">About</a>
+                <a href="#">Advertise</a>
+                <a href="#">Jobs</a>
+                <a href="#">Contact</a>
+            </div>
+        </div>
+
+        <a class="navigation-logo" href="#">
+                <img class="top-nav-tnw-logo" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="tnw logo">
+        </a>
```

Comment:
- [ ] Check the size of the logo in the tablet and mobile versions. It's not behaving correctly.
![Screenshot from 2020-03-18 16-19-01](https://user-images.githubusercontent.com/51926937/77012993-b985f000-6934-11ea-9604-74afa007c874.png)
![Screenshot from 2020-03-18 16-19-16](https://user-images.githubusercontent.com/51926937/77012996-bab71d00-6934-11ea-862e-78fedd8b01fb.png)